Servers/UI/OJS.Servers.Ui/ClientApp/src/styles/_variables.scss

```diff
@@ -6,3 +6,29 @@ $f-size-18: 18px;
 $f-size-24: 24px;
 $f-size-30: 30px;
 $f-size-36: 36px;
+
+$f-size-1vw: 1vw;
+$f-size-2vw: 2vw;
+
+// Responsive width breakpoints
+$r-mobile-min: 320px;
+$r-mobile-max: 480px;
+
+$r-tablet-min: 481px;
+$r-tablet-max: 768px;
+
+$r-small-min: 769px;
+$r-small-max: 1024px;
+
+$r-large-min: 1025px;
+$r-large-max: 1200px;
+
+$r-small-desktop-min: 1201px;
+$r-small-desktop-max: 1600px;
+
+$r-large-desktop-min: 1601px;
+$r-large-desktop-max: 1920px;
+
+$r-extra-large-desktop-min: 1921px;
```

Comment:
It is better to use cascading media queries, i.e. use only `min-width`, or only `max-width`, but not both of them.
Correct examples:
```
@media only screen and (max-width: 1920px) // desktop-first
@media only screen and (min-width: 1440px) // mobile-first
```

Incorrect examples:
```
@media only screen and (min-width: variables.1440px) and (max-width: 1920px)
```

Since the application is not build mobile-first, use only `max-width`